doc/radosgw/config-ref.rst

```diff
@@ -7,6 +7,11 @@ The following settings may added to the Ceph configuration file (i.e., usually
 settings may contain default values. If you do not specify each setting in the
 Ceph configuration file, the default value will be set automatically.
 
+Configuration variables set under the ``[client.radosgw.{instance-name}]``
+section will not apply to rgw or radosgw-admin commands without an instance-name
+specified in the command. Thus variables meant to be applied to all RGW
+instances or all radosgw-admin commands can be put into the ``[global]`` or the
```

Comment:
@tchaikov my rationale behind adding that sentence is many more variables besides `rgw override bucket index max shards` and `rgw bucket default quota max objects` could be added to the `[global]` section to be applied for all instances of the radosgw-admin command. This section is there to make people reading the docs mindful of that and make them think twice before adding variables to the `[client.radosgw.{instance-name}]` section.